src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -210,21 +211,32 @@ const Conversation: React.FC<ConversationProps> = props => {
   })
 
   return (
-    <Flex flexDirection="column" width="100%">
-      <Box>
-        <Spacer mt={["75px", 2]} />
-        <Flex flexDirection="column" width="100%" px={1}>
-          {inquiryItemBox}
-          {messageGroups}
-          <Spacer mb={[undefined, 6]} />
-          <Box ref={bottomOfPage}></Box>
-        </Flex>
-      </Box>
+    <NoScrollFlex flexDirection="column" width="100%">
+      <MessageContainer>
+        <Box>
+          <Spacer mt={["75px", 2]} />
+          <Flex flexDirection="column" width="100%" px={1}>
+            {inquiryItemBox}
+            {messageGroups}
+            <Box ref={bottomOfPage}></Box>
+          </Flex>
+        </Box>
+      </MessageContainer>
       <Reply conversation={conversation} environment={relay.environment} />
-    </Flex>
+    </NoScrollFlex>
   )
 }
 
+const MessageContainer = styled(Box)`
+  height: calc(100% - 300px);
+  flex-grow: 1;
+  overflow-y: scroll;
+`
```

Comment:
So this `MessageContainer` has its height bound to `100%` minus things above and below it. The scroll happens now happens in this box and the parent (`NoScrollFlex`) doesn't do scroll. `flex-grow: 1` makes the `MessageContainer` to take the entire height available to it so it guarantees that the reply box always appears in the bottom of the page.

Reply:
Looks like this helps with some of those extra long messages as well. Now you can have access to those messages that were getting hidden.


![Screen Recording 2020-07-06 at 12 59 58 PM](https://user-images.githubusercontent.com/12748344/86619977-87eb7000-bf89-11ea-99f2-7329b6c99525.gif)